src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java

```diff
@@ -32,10 +33,23 @@ public class VoltageLevelMapData {
 
     private double nominalVoltage;
 
+    private double lowVoltageLimit;
+
+    private double highVoltageLimit;
+
+    private Double ipMin;
+
+    private Double ipMax;
+
+    private int busbarCount;
```

Comment:
@JsonInclude(JsonInclude.Include.NON_NULL)
private Integer busbarCount;

same for other int fields